### ISSUE TITLE
VCAL for 240 AC is wrong

### ISSUE THREAD

**Issue by MohammedBENHADINE** [mcmchris/MCM-Meter-2.0#1]
First thanks for this incredible project !

After checking the code it seems that you are using VCAL = 164.7 ( for 240 AC ) , but based on the schematic the initial calibration value is (( 240v*110kOhm) / (9v * 10 kOhm )) which is around 293.33.

Could you please explain why you have chosen 164.7 instead of 293.33 ?

Thanks

**Comment by mcmchris** [comment]
Hi, Thanks

You are right, first I configured my project to be connected between both live lines (L1 & L2 - 240v) but setting the VCAL parameter manually to set my Voltage lecture to 120V because it's my house voltage in Latin America, I did that to make lectures more precise, so if any line fluctuate this affect the entire voltage measurement.

While testing with my oscilloscope I noticed that my ADC was saturated when connected to 240v (L1 & L2) directely because of my PT relation (240v-9v) so I considered connecting just to L1 and Neutral to avoid saturation but considering just one line.

The parameter is wrong because of that.
